Gemfile

```diff
@@ -28,33 +57,12 @@ gem 'refills'
 gem 'rest-client', '1.8.0'
 gem 'awesome_print'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.3'
-# Use postgresql as the database for Active Record
-gem 'pg', '0.18.2'
-# Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.1.0'
-# See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
+# Allow user to easily switch users in dev
+gem "switch_user", group: :development
```

Comment:
`switch_user` along with new seed means we can easily switch users in dev. No more signing in with real credentials (which is a very bad idea and should be fixed for all our apps.)